roles/cert_manager/tasks/validate_certs.yml

```diff
@@ -21,11 +21,14 @@
     mode: '0755'
 
 - name: Install cert-manager cmctl CLI
-  ansible.builtin.unarchive:
-    src:
-      "https://github.com/cert-manager/cert-manager/releases/{{ cifmw_cert_manager_version }}/download/cmctl-linux-amd64.tar.gz"
-    dest: "{{ lookup('env', 'HOME') }}/bin/"
-    remote_src: true
+  become: true
+  vars:
+    _os: "{{ lookup('pipe', 'go env GOOS') }}"
```

Comment:
humpf, we need to mock that lookup I think - the pod doesn't have `go` - hence the lookup is failing. At least that's my understanding of the issue. it means editing `.ansible-lint` to extend the mocked modules (?) by adding
`lookup_plugin.pipe`...